packages/ui/src/primitives/tailwind/LoadingView/index.tsx

```diff
@@ -89,7 +89,7 @@ const LoadingView = ({
       ) : (
         loader
       )}
-      {title && <Text className={titleClassname}>{title}</Text>}
+      {title && <Text className={twMerge(titleClassname, 'text-text-primary')}>{title}</Text>}
```

Comment:
we are using tailwind progress for the uploading progress. any reason why we added/changed this?